css/style.css

```diff
@@ -542,6 +542,11 @@ summary:hover .summary-chevron-up svg {
   margin-top: 18px;
 }
 
+.error-message {
+  color: rgb(185, 14, 14);
+  display: none;
+}
+
```

Comment:
- [ ] The error message should show up close to the "submit" button. Kindly implement it.